packages/bindgen/src/realm_js_helpers.h

```diff
@@ -265,6 +266,16 @@ struct Helpers {
     {
         ws.feed_buffer({buffer.data(), buffer.size()});
     }
+
+    static auto make_ssl_verify_callback(std::function<bool(const std::string& server_address, int server_port,
+                                                            std::string_view pem_data, int preverify_ok, int depth)>
+                                             callback)
+    {
+        return [callback = std::move(callback)](const std::string& server_address, uint16_t server_port,
+                                                const char* pem_data, size_t pem_size, int preverify_ok, int depth) {
+            return callback(server_address, server_port, std::string_view(pem_data, pem_size), preverify_ok, depth);
+        };
+    }
```

Comment:
1. [signature]: Take the JS-wrapped version of the callback as the argument. This callback will (when invoked) call the user-provided callback for validating certificates.
2. [1st `return`]: Return the CPP (Core-friendly) version of the callback (stored in the binding).
3. [2nd `return`]: When the CPP callback is invoked, invoke the JS-wrapped callback and return its result.